server/resolvers/index.spec.ts

```diff
@@ -29,6 +30,108 @@ test('createQuiz returns an id', () => {
 });
 
 
+test('launchNextQuestion should return true if there is a question', () => {
+    const quizId = resolvers.Mutation.createQuiz();
+    const quiz = resolvers.Mutation.updateQuiz(undefined, {
+        input: {
+            id: quizId,
+            name: 'MyNewQuiz',
+            questions: [
+                {question: 'Which day was on Jan, 1st, 2019?', answers: []}
+            ]
+        }
+    });
+
+    const result = resolvers.Mutation.launchNextQuestion(undefined, {operatorId: quiz.operatorId});
+    expect(result).toBe(true);
+});
+
+test('launchNextQuestion should return false if there is no further question', () => {
+    const quizId = resolvers.Mutation.createQuiz();
+    const quiz = resolvers.Mutation.updateQuiz(undefined, {
+        input: {
+            id: quizId,
+            name: 'MyNewQuiz',
+            questions: []
+        }
+    });
+
+    const result = resolvers.Mutation.launchNextQuestion(undefined, {operatorId: quiz.operatorId});
+    expect(result).toBe(false);
+});
+
+test('launchNextQuestion should trigger NEXT_QUESTION', (done) => {
+    const quizId = resolvers.Mutation.createQuiz();
+    const quiz = resolvers.Mutation.updateQuiz(undefined, {
```

Comment:
Ich würde die Mockdaten in eine Konstante extrahieren, da sie mehrmals verwendet werden.

Reply:
done